### PR TITLE
haskell: Update author name

### PR DESCRIPTION
**Pull request by ghost**
I updated my Github username, and some bot is now parked at the old username. I figured I should open a PR to update this.

Referring to the PRs (which show _this_ updated username): https://github.com/zed-industries/zed/pull/6786, https://github.com/zed-industries/zed/pull/6995, https://github.com/zed-industries/zed/pull/7331.

Release Notes:

- N/A

### CODE REVIEW

File: extensions/haskell/extension.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description = "Haskell support."
 version = "0.1.0"
 schema_version = 1
 authors = [
-    "Pseudomata <pseudomata@proton.me>",
+    "Pocæus <github@pocaeus.com>",
```
Comment (pocaeus) [inline extensions/haskell/extension.toml:7]: can s/`æ`/`ae` if special characters cause issues.

**Comment by pocaeus** [comment]
@cla-bot check

File: extensions/haskell/extension.toml (hunk 1 above)
Comment (maxdeviant) [reply extensions/haskell/extension.toml:7]: Looks like it should render fine:

Screenshot 2024-06-19 at 10 28 06 AM

**Review by maxdeviant — APPROVED** [review]
Thanks!

**Pull request merged**